src/web.ts

```diff
@@ -289,15 +289,15 @@ export class FirebaseAnalyticsWeb extends WebPlugin
         document.getElementById(scripts[0]) &&
         document.getElementById(scripts[1])
       ) {
-        return resolve();
+        return resolve( null );
```

Comment:
Instead of passing a value where none is expected, can you modify the promise to declare a type, _e.g._:
```
return new Promise<void>(async (resolve, _reject) => {
```